### PR TITLE
Added the following recommendations to Azure Site Recovery.

### PR DESCRIPTION
**Pull request by aalewismsft**
1. ASR-3 - Monitor Source VMs for high data change rates
2. ASR-4 - Ensure sufficient resource quotas in the target region

# Overview/Summary

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## Related Issues/Work Items

Replace this with a list of related GitHub Issues and/or ADO Work Items (Internal Only)

- To associate a GitHub Issue, use a [key word](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) preceded with the GitHub issue number.
- To associate an ADO Work Item, use the key word `AB#` succeeded with the [ADO Work Item ID](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## This PR fixes/adds/changes/removes

1. *Replace me*
2. *Replace me*
3. *Replace me*

### Breaking Changes

1. *Replace me*
2. *Replace me*

## As part of this Pull Request I have

- [ ] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [ ] Ensured PR tests are passing
- [ ] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: docs/content/services/compute/site-recovery/_index.md [hunk 1]
```diff
@@ -75,3 +75,49 @@ Test your Disaster Recovery plan periodically without any data loss or downtime,
 {{< /collapse >}}
 
 <br><br>
+
+### ASR-3 - Monitor Source VMs for high data change rates
```
Comment (oZakari) [inline docs/content/services/compute/site-recovery/_index.md:79]:
```suggestion
### ASR-3 - Monitor source VMs for high data change rates
```

File: docs/content/services/compute/site-recovery/_index.md [hunk 2]
```diff
@@ -75,3 +75,49 @@ Test your Disaster Recovery plan periodically without any data loss or downtime,
...
+
+**Category: Disaster Recovery**
+
+**Impact: High**
+
+**Guidance**
+
+Azure Site Recovery has limits on data change rates, depending on the type of disk. To see if this problem is recurring or temporary, find the data change rate of the affected virtual machine. Go to the source virtual machine and monitor the **OS Disk Write Bytes/Sec** and **Data Disk Write Bytes/Sec** metrics for OS and Data disks.
```
Comment (oZakari) [inline docs/content/services/compute/site-recovery/_index.md:87]:
```suggestion
Azure Site Recovery has limits on data change rates, depending on the type of disk. To determine if this is a recurring or temporary problem, find the data change rate of the affected virtual machine. Go to the source virtual machine and monitor the **OS Disk Write Bytes/Sec** and **Data Disk Write Bytes/Sec** metrics for OS and Data disks.
```

File: docs/content/services/compute/site-recovery/code/asr-3/asr-3.kql
Comment (oZakari) [inline docs/content/services/compute/site-recovery/code/asr-3/asr-3.kql:1]: Please review this contribution guidance to make sure your query adheres to all requirements: https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing/#azure-resource-graph-arg-queries

File: docs/content/services/compute/site-recovery/_index.md
Comment (oZakari) [inline docs/content/services/compute/site-recovery/_index.md:1]: As these are new recommendations, you will also need to update the table toward the top of this page.

**Review by oZakari — CHANGES_REQUESTED** [review]
Hi @aalewismsft, thanks for your additions to ASR! Please go through the comments below and complete the PR form. Once finished, feel free to request another review from me.

**Review by aalewismsft — COMMENTED** [review]
Requested changes implemented.

File: docs/content/services/compute/site-recovery/_index.md [hunk 3]
```diff
@@ -75,3 +77,52 @@ Test your Disaster Recovery plan periodically without any data loss or downtime,
...
+If the target location has a capacity constraint, disable replication to that location. Then, enable replication to a different location where your subscription has sufficient quota to create VMs of the required sizes.
+
+**Resources**
+
+- [Azure resource quota issues](https://learn.microsoft.com/en-us/azure/site-recovery/azure-to-azure-troubleshoot-errors#azure-resource-quota-issues-error-code-150097)
+
+<br><br>
+
+
+
```
Comment (oZakari) [inline docs/content/services/compute/site-recovery/_index.md:128]:
```suggestion
```

File: docs/content/services/compute/site-recovery/code/asr-3/asr-3.kql
Comment (oZakari) [reply docs/content/services/compute/site-recovery/code/asr-3/asr-3.kql:1]: Hi @aalewismsft, could you review this query again and the link above to make sure that the required column names are included as well as any additional parameters show up as param1, param2, .... up to param5? 

Also, please update the PR form and include a screenshot of the query results.
image

**Pull request closed without merging**